github-issues/js/src/index.ts

```diff
@@ -3,13 +3,32 @@ import * as zlib from "zlib";
 import * as util from "util";
 import axios from "axios";
 
-import { ChatCompletionRequestMessage, Configuration, OpenAIApi } from "openai";
+import {
+  ChatCompletionRequestMessage,
+  Configuration,
+  CreateChatCompletionRequest,
+  OpenAIApi,
+} from "openai";
 import * as sqlite3 from "sqlite3";
 import * as braintrust from "braintrust";
+import throttledQueue from "throttled-queue";
 
 const CACHE_PATH = "./cache";
 fs.mkdirSync(CACHE_PATH, { recursive: true });
 
+function checkEnvVar(name: string) {
+  if (!process.env[name]) {
+    console.log(
+      `Please set the ${name} environment variable to run this script.`
+    );
+    process.exit(1);
+  }
+}
+
+checkEnvVar("OPENAI_API_KEY");
+checkEnvVar("BRAINTRUST_API_KEY");
+checkEnvVar("BRAINTRUST_API_URL");
```

Comment:
users don't need to set this. Only we do because we're running locally (not through braintrustdata.com).

Reply:
Won't they have their custom domains for that?